packages/cli/cli/src/commands/validate/validateDocsWorkspaceAndLogIssues.ts

```diff
@@ -1,44 +1,53 @@
 import { APIWorkspaceLoader, validateDocsWorkspace } from "@fern-api/docs-validator";
 import { TaskContext } from "@fern-api/task-context";
-import { DocsWorkspace } from "@fern-api/workspace-loader";
+import { DocsWorkspace, FernWorkspace } from "@fern-api/workspace-loader";
 
 import { logViolations } from "./logViolations";
 
 export async function validateDocsWorkspaceWithoutExiting({
     workspace,
-    loadAPIWorkspace,
+    fernWorkspaces,
     context,
     logWarnings,
+    errorOnBrokenLinks,
     logSummary = true
 }: {
     workspace: DocsWorkspace;
-    loadAPIWorkspace: APIWorkspaceLoader;
+    fernWorkspaces: FernWorkspace[];
     context: TaskContext;
     logWarnings: boolean;
+    errorOnBrokenLinks?: boolean;
     logSummary?: boolean;
 }): Promise<{ hasErrors: boolean }> {
-    const violations = await validateDocsWorkspace(workspace, context, loadAPIWorkspace);
-    const { hasErrors } = logViolations({ violations, context, logWarnings, logSummary });
+    const violations = await validateDocsWorkspace(workspace, context, fernWorkspaces);
+    let { hasErrors } = logViolations({ violations, context, logWarnings, logSummary });
+
+    if (errorOnBrokenLinks) {
+        hasErrors = hasErrors || violations.some((violation) => violation.name === "valid-markdown-links");
```

Comment:
It might be worth updating the `valid-markdown-links` name to to `no-broken-links` here so that the flag matches the name upfront. But we can do this in a follow-up if you prefer.

Reply:
yeah was planning to do this as a follow-up